spark-jobs/src/main/scala/filodb/downsampler/chunk/DownsamplerMain.scala

```diff
@@ -98,19 +98,14 @@ class Downsampler(settings: DownsamplerSettings) extends Serializable {
       index.get
     })
 
-    val saveAddress = batchExporter.makeExportAddress(exportKey)
+    val tablePath = batchExporter.makeExportAddress(exportKey)
     val filteredRowRdd = rdd.flatMap(batchExporter.getExportRows(_)).filter{ row =>
       val rowKey = rowKeyIndices.map(row.get(_).toString)
       rowKey == exportKey
     }
+    // convert filteredRowRdd to df and append to iceberg table
+    batchExporter.writeDataToIcebergTable(sparkSession, settings, tablePath, exportKey(0), filteredRowRdd)
```

Comment:
Instead of `exportKey(0)`, could we use something like this to build a table name from the export key?
```
table-format = "$0"
```

Reply:
https://github.com/filodb/FiloDB/pull/1718/commits/8efc7c9f8ca132c6a48a62f4801373cb5330bc15

Reply:
Implemented export table config for export keys under groups to specify table and table-path for every group to be exported.